apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -224,7 +240,7 @@ export default function ListBox({
       itemCount={count}
```

Comment:
I think this also should be using listCount
```suggestion
      itemCount={listCount}
```

Reply:
Yes it should.

Reply:
Unfortunately this does not work in the DQ scenario I've tested. When we perform selections in a field which does not have selections from before – previously excluded values (`qState == 'X'`) are not filtered anymore. In other words: X + S (in same field) => X-values are not filtered.

When we clear the selections of the field the Loader does not ask for more data than it currently has knowledge of (which is based on current pages). Maybe it's possible to send an additional request to find this out but I we should keep `qcy` for now.

Reply:
For that case should not 'getCalculatedHeight' detect that there is no filtering anymore and return count that is `qcy`?